Documentation/contributing/testing/ci.rst

```diff
@@ -153,7 +153,9 @@ packet.net before the branch is merged.
 Testing matrix
 ^^^^^^^^^^^^^^
 
-We are currently testing following kernel - k8s version pairs in our CI:
+Note: depending on the time at which it was built, the documentation below might
+be outdated. Up to date CI testing information regarding k8s - kernel version
+pairs can always be found in the `Cilium CI matrix`_.
```

Comment:
If we're saying this information will rot away quickly, maybe we should just remove it and rely only on the Cilium CI matrix. Otherwise, we should at least update the K8s version for GKE.

EDIT: I think keeping the docs for trigger phrases below makes sense because they should change less often (compared to Kubernetes versions).

Reply:
I agree, I actually was writing a comment on that so might as well put it here: at first I intended to remove the versions table completely, but as I was writing the doc I was thinking it was also nice to keep it as-is for historical purposes. I.e. someone browsing docs for 1.10 in the future may want to know against which type of stack this version was tested.

At the present time it is redundant with the GSheet, but I was unsure if we would keep documentation for deprecated Cilium versions online for longer than the time their gets removed from the GSheet. What's your take on this?

Reply:
How about we link to the GSheet from the docs?

Historic support should be available from this page: https://docs.cilium.io/en/v1.9/concepts/kubernetes/requirements/

Reply:
Yea, the GSheet has already been added to the docs in this PR, so my question was do we want both or only the GSheet -- though I guess you implicitly said that the latter is sufficient :)

Reply:
> though I guess you implicitly are saying that the latter is sufficient.

:+1:

Reply:
Removed table completely.